### DISCUSSION TITLE
Welcome to openMVG Discussions!

### DISCUSSION THREAD

**Discussion by pmoulon** [openMVG/openMVG#1832] [General]
## 👋 Welcome to OpenMVG!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Wonder how to start with OpenMVG:
- [Wiki](https://github.com/openMVG/openMVG/wiki)
- [Online doc](https://openmvg.readthedocs.io/en/latest/)
- [3D reconstruction Bibliography](https://github.com/openMVG/awesome_3DReconstruction_list)  Please help to keep it up to date
- [Dataset](https://github.com/openMVG/Image_datasets)

**Comment by RahulDas-dev** [comment]
Hi Everyone,

I am trying to implement incrementalSFM on 20 images, with EXIF metadata [Latitude, Longitude]. I know the next step is matching, but want to go with LOW ratio match on 20 images and then rest of 19. 

I was wondering if there was a way to intelligently determine the probable match by using the exif data or any other method .
Or 
how **OpenMVG** Solve this Problem ??

Thank you in advance

**Reply by pmoulon** [reply]
Note that we have a matching that use pair coming form GPS neighborhood position
See here https://github.com/openMVG/Image_datasets/tree/main/Drone#using-gps-prior-for-pair-selection-and-sfm-prior-optimize-in-gps-xyz-coordinates
You even have a dataset to play with ;-)

**Reply by RahulDas-dev** [reply]
Thanks

**Comment by JackyXiao1515** [comment]
Hi everyone!

I'm trying to build up the OpenMVG on my Windows computer. I followed the instructions as "build.md" says, but after I cloned the code and configured in CMake, I found there are a lot values are NOTFOUND.

e.g. 
CLPSOLVER_LIBRARY-NOTFOUND, 
CLP_LIBRARY-NOTFOUND, 
Ceres_DIR-NOTFOUND, 
Eigen3_DIR-NOTFOUND, 
GHOSTSCRIPT_EXECUTABLE-NOTFOUND
![NOTFOUND](https://github.com/openMVG/openMVG/assets/100830070/4756d7b0-b619-4d2e-8e2a-2b3403a4342f)

Is it means I still need to download extra packages? Or what can I do to fix this problem?

Best regards

**Comment by pmoulon** [comment]
Did you checkout with recursive or have initialized the submodule?

I rarely use the cake gui interface so seeing some value empty or not found could maybe normal
